db/migrate/20190930192109_add_using_openstax_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsingOpenStaxToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :using_openstax, :string
```

Comment:
I think `using_openstax` should be a boolean instead of a string that stores "Yes" or "No".

Reply:
If I made it a boolean, it would need to have a default value. Dante suggested I not have a default value so it only shows up in the API when data is available.

Reply:
By this "it only shows up in the API when data is available" — you mean the field `using_openstax` entirely, or the value of the field so when no data is available its value is `nil`?

Because if you mean the value of the field only shows up when data is available, then the value in that case could be `nil`, and if you mean the field `using_openstax` itself, then that's weird that it only shows some times. Keep in mind that `nil` ("no data available") can be represented as `null` in JSON.

Reply:
** I see that you do mean the field's value.

So yeah, I think we can still make it a boolean and represent "No data available`, `nil` in Ruby, as `null` in JSON.